packages/office-ui-fabric-react/src/components/Rating/Rating.base.tsx

```diff
@@ -96,33 +101,35 @@ export class RatingBase extends BaseComponent<IRatingProps, IRatingState> {
       theme: theme!
     });
 
-    for (let i = min as number; i <= (max as number); i++) {
-      const ratingStarProps: IRatingStarProps = {
-        fillPercentage: this._getFillingPercentage(i),
-        disabled: disabled ? true : false,
-        classNames: this._classNames
-      };
-
-      starIds.push(this._getStarId(i - 1));
-
-      stars.push(
-        <button
-          className={ css(this._classNames.ratingButton, {
-            [this._classNames.rootIsLarge]: size === RatingSize.Large,
-            [this._classNames.rootIsSmall]: size !== RatingSize.Large
-          }) }
-          id={ starIds[i - 1] }
-          key={ i }
-          { ...((i === Math.ceil(this.state.rating as number)) ? { 'data-is-current': true } : {}) }
-          onFocus={ this._onFocus.bind(this, i) }
-          disabled={ disabled || readOnly ? true : false }
-          role='presentation'
-          type='button'
-        >
-          { this._getLabel(i) }
-          <RatingStar key={ i + 'rating' }  {...ratingStarProps} />
-        </button>
-      );
+    for (let i = this._min as number; i <= (max as number); i++) {
+      if (i !== 0) {
```

Comment:
I'm confused at this. Won't this totally not render any stars for 0? Meaning, we want the BUTTON to be there, just not lit up as if it has a value..

Reply:
We need it to register zero as a valid value (meaning rating can equal zero), but when it loops from 0 to 5, we want it to print out only 5 stars, skipping the "lowest value" if that is zero:

 0 - 1 - 2 - 3 - 4 - 5
🛇  ★   ★  ★   ★  ★

Also, if you are wondering why we don't just loop from 1 to 5, that is because the min can be value 1 or 0.  If it's 1 we want to print the star, if it's specifically 0, we want to skip printing that one.